config/crd/patches/webhook_in_spacetemplates.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: spacetemplates.nauticus.io.nauticus.io
```

Comment:
This should be updated. We have made a mistake when we generated the kubebuilder project.